dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebAppConfiguration.java

```diff
@@ -2169,4 +2169,38 @@ public void setModuleResponseEncoding(String encoding) {
         }
         this.responseEncoding = encoding;
     }
+    
+    /**
+     * Since Servlet 6.0 - opt-out verifying the encoded char in URI.
+     * 
+     * true - skip checking for %23 , %2e , %2f , %5c in URI
+     */
+    public void setSkipEncodedCharVerification() {
+        if (this.contextParams != null){
+            String value = (String) this.contextParams.get("SKIP_ENCODED_CHAR_VERIFICATION");
+            if (value != null){
+                if(value.equalsIgnoreCase("true")){
+                    if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE))
+                        logger.logp(Level.FINE, CLASS_NAME,"setSkipEncodedCharVerification", "SKIP verifying encoded character in URI for application -> "+ applicationName);
+                    this.skipEncodedCharVerification = true;
+                }
+                else{ // false for either invalid/false
+                    if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE))
+                        logger.logp(Level.FINE, CLASS_NAME,"setSkipEncodedCharVerification", "VERIFY encoded character in URI for application -> "+ applicationName);
+                }
+
+                return;
+            }
+        }
+
+        if (WCCustomProperties.SKIP_ENCODED_CHAR_VERIFICATION){
+            if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE))
+                logger.logp(Level.FINE, CLASS_NAME,"setSkipEncodedCharVerification", "via server property. SKIP verifying encoded character in URI for application -> "+ applicationName);
+            this.skipEncodedCharVerification = true;
+        }
+    }
+
+    public boolean skipVerifyEncodedCharInURI() {
```

Comment:
Since we're returning the value of this variable a better method name would be `isSkipVerifyEncodedCharInURI`

Reply:
change to isSkipVerifyEncodedCharInURI